feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/thresold/gov2/curve/ReciprocalCurve.kt

```diff
@@ -23,11 +23,11 @@ class ReciprocalCurve(
     override val name: String = "Reciprocal"
 
     override fun threshold(x: Perbill): Perbill {
-        return factor.divide(x + xOffset, MathContext.DECIMAL64) + yOffset
+        return factor.divide(x + xOffset, MathContext(MathContext.DECIMAL64.precision, RoundingMode.DOWN)) + yOffset
     }
 
     override fun delay(y: Perbill): Perbill {
-        val maybeTerm = factor.divideOrNull(y - yOffset, MathContext(MathContext.DECIMAL64.precision, RoundingMode.HALF_UP))
+        val maybeTerm = factor.divideOrNull(y - yOffset, MathContext(MathContext.DECIMAL64.precision, RoundingMode.UP))
 
```

Comment:
Fixed rounding according to substrate sdk: https://github.com/paritytech/substrate/blob/37664fe5b3513eb996225f016eceaf74963b8133/frame/referenda/src/types.rs#L450